lib/npm-util.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const execa = require('execa');
+const pTimeout = require('p-timeout');
+
+exports.connectToNpm = () => pTimeout(
+	(async () => {
+		try {
+			await execa('npm', ['ping']);
+			return true;
+		} catch (_) {
+			throw new Error('Connection to npm registry failed');
+		}
+	})(),
+	15000,
+	'Connection to npm registry timed out'
+);
+
+exports.npmUsername = async () => {
+	try {
+		return await execa.stdout('npm', ['whoami']);
+	} catch (error) {
+		return error;
+	}
+};
+
+exports.npmCollaborators = async pkgName => {
```

Comment:
Also use `ow` here to ensure `pkgName` is defined and a string.

Reply:
> Also use `ow` here to ensure `pkgName` is defined and a string.

Shouldn't this be done in a more general context such as in `prerequisite.js`?

Reply:
No, this is just about ensuring the function is called with correct arguments. It might be used on other places later on. This is not ensuring the `package.json` is correct. This is just checking that this exact function is called correctly.